piptools/scripts/compile.py

```diff
@@ -7,10 +7,7 @@
 import click
 from click.utils import safecall
 from pip._internal.commands import create_command
-from pip._internal.req.constructors import (
-    install_req_from_line,
-    install_req_from_req_string,
-)
+from pip._internal.req.constructors import install_req_from_line
```

Comment:
I remember struggling for hours to understand the difference between these two functions, and it seems that I still didn't fully get them... thanks @atugushev 👍🏽